packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4966,6 +4966,7 @@ export default class Meeting extends StatelessWebexPlugin {
         case ConnectionState.Connected:
           NewMetrics.submitClientEvent({
             name: 'client.ice.end',
+            payload: {icePhase: 'IN_MEETING'},
```

Comment:
this code gets executed both during join and in-meeting, it's not going to be straightforward to determine which case is it (we'll probably have to add some flag to the meeting object to know when we connect successfully for the first time), so maybe we can just skip sending icePhase here? @torgeadelin what do you think? does CA team care about this? 
if we want do set it properly, it needs to be 'JOIN_MEETING_FINAL' if this is the first time we are "connected" for the current meeting join attempt and 'IN_MEETING' for any other case

Reply:
Not sure if you mean skipping sending ice.end entirely? I know that we should always send ice.start and ice.end.

Reply:
I think Marcin mean not sending icePhase in this particular case

Reply:
what does this mean tho? was there a client.ice.end sent previously if we reach this case?

Reply:
> was there a client.ice.end sent previously if we reach this case?

no, not necessarily. This same piece of code is executed when we successfully connect the first time or at any later point after a brief network loss